app/test_main.py

```
@@ -1,3 +1,27 @@
+from typing import List
+
+
+import pytest
+
+
 from app.main import get_human_age
 
+
 # write your code here
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ],
+)
+def test_get_human_age(
+        cat_age: int, dog_age: int, expected: List[int]) -> None:
+    """Test get_human_age for cats and dogs with multiple scenarios."""
+    assert get_human_age(cat_age, dog_age) == expected
```

This is a good set of test cases, but it's missing some important edge cases required by the task's checklist.

This violates checklist item #2: "Make sure you've tests checks all edged situations such as: ... if function resive data out of normal range, such as negative numbers, zero, or realy large numbers; if the function receives an incorrect type of data it raises the correct exception."

You should add tests to cover:
- **Negative numbers**: For example, `get_human_age(-1, -5)`.
- **Incorrect data types**: What happens if you pass a string like `'ten'`? The function should raise an exception, and your test should verify this. You can use `pytest.raises` to test for exceptions.